dbs/mongo/queries/getQuestionData.js

```diff
@@ -0,0 +1,26 @@
+const {Questions, Answers, Answer_Photos} = require('../models.js');
+const {formatResponse} = require('../controllers/questionResponse.js');
+
+async function getQuestionData(p_id, page, count) {
+
+  let questions = await Questions.find({product_id:p_id, reported: 0}, {_id: 0}).limit(count);
+
+  let answers = await Promise.all(questions.map(async (question) => {
```

Comment:
Is it necessary to use Promise.all when you are using async - await? And where those promises get resolved / rejected?